### ISSUE TITLE
Not sure what I am doing wrong here but I cannot disable voting message

### ISSUE THREAD

**Issue by LadyAnimeYuki** [BenCodez/VotingPlugin#1124]
1.15.2 spigot

I was having issues with the plugin giving double rewards for every vote and double messages. I was able to get that to stop ( it throws an error in the console but I would rather that than double rewards ). I have disabled everywhere I can find "Thanks for voting for us on xsite, you have been rewarded with $1000." The only reason I want this disabled is because immediately after you get the eco message saying 1k was added to your balance. It floods the screen a little. I tried putting a "Thanks for voting" message on the rewards part but it doesn't even show just the first message about which site and their reward. I also had some issues with the vote reminder flooding my chat even though I have it set to false. I had to go in and turn off the permissions to be able to see that to make it stop. =( 
Just tried again and now I'm getting two of the thank you messages and I have no idea why, ( one appears to be broadcasting to the entire server which I do not want enabled ) also one of the links doesn't let me click it, believe it is the ServerlistGames one, as well as regular players can't always open the gui even with the right permissions but can do /voteall 
In short. This is a mess and I know it's on my end but any help is greatly appreciated Thank you in advance
config file: apparently dont use pastebin.pl because its cancer for some unknown reason.
votesites: redacted

**Comment by BenCodez** [comment]
use pastebin.com, pastebin.pl is an unsafe website. 

Are you using a bungee setup?

**Comment by LadyAnimeYuki** [comment]
Was unaware of that sorry and no it's just a single basically vanilla world set up so there isn't really a need for bungee as far as I'm aware.

**Comment by BenCodez** [comment]
Wanna send your votesites.yml via pastebin.com?

**Comment by LadyAnimeYuki** [comment]
https://pastebin.com/pVsRVmis

**Comment by BenCodez** [comment]
Under each votesite add Rewards: []

That might do it.

**Comment by LadyAnimeYuki** [comment]
Still getting https://i.gyazo.com/05245cb390180f39338115ead74a3e7d.png as a result.

**Comment by BenCodez** [comment]
The second message if I had to guess might be the broadcast in config.yml

**Comment by LadyAnimeYuki** [comment]
Right but I have that set to  
# Broadcast vote message
# Set to an empty message to disable
  BroadcastMsg: ''
( no idea why github wants to make this huge and bold sorry about that )

**Comment by BenCodez** [comment]
Did you set anything in specialrewards.yml? If not it might be another plugin doing that. You can always set debuglevel to EXTRA in the config and send the log on a vote. It should pinpoint the exact sources.

**Comment by LadyAnimeYuki** [comment]
I did not change anything in specialrewards.yml. I will do a debuglevel on extra when I'm home later thank you.

**Comment by LadyAnimeYuki** [comment]
Attempted a EXTRA debug but nothing showed up while attempting the commands, all it said was I didn't have permissions. I made sure that luckperms is allowing the players to use the commands but when they do it says they do not have permissions. They used to be able to do /voteall but now it is a blank list. I'm now getting more constant reminders to vote even though I have all those settings to false and even turned the permissions off in luckperms. It is spamming my screen and I've spent over 15 hours attempting to configure this. I really wanted to use this plugin but it literally will not let me. I don't know why anything I do seems to be overwritten by some unseen file somewhere. I'm still getting the broadcast messages even though that is also set to be empty.

**Comment by BenCodez** [comment]
You reloaded the plugin or even restarted after setting debug? It should be almost spamming if you did.

**Comment by LadyAnimeYuki** [comment]
Got it figured out thank you for your time.
